modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -61,19 +65,26 @@ void OnAmxxAttach()
 		return;
 	}
 
-	void *address = nullptr;
-
-	if (!CommonConfig->GetAddress("g_pGameRules", &address) || !address)
+	if ((HasRegameDll = RegamedllApi_Init()))
 	{
-		MF_Log("get/set_gamerules_* natives have been disabled because g_pGameRules address could not be found. ");
-		return;
+		GameRulesRH = ReGameApi->GetGameRules();
```

Comment:
Unless I'm wrong, this should be fine. `AMXX_Attach` is called at world spawn, and `g_pGameRules` is initialized at world precache, so it should not be null. I think I've tested and I did want to add more code than necessary.